mobile/client/src/screens/Registration.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import {StyleSheet, Text, TextInput, TouchableOpacity, View} from 'react-native';
+
+class Reg extends React.Component {
+    constructor() {
+    super();
+    this.state = {
+      username: "",
```

Comment:
Should we change to "nickname"? That is what the web app uses as of rn

Reply:
Could change the web app to username also

Reply:
Oh yeah, if we want to keep things consistent then that makes sense.